command.go

```diff
@@ -51,7 +51,19 @@ func (o *Command) addArg(a *arg) error {
 		current = current.parent
 	}
 	a.parent = o
+
+	if a.GetPositional() {
+		switch a.argType {
+		case Flag, StringList, IntList, FloatList, FileList:
+			return fmt.Errorf("argument type cannot be positional")
+		}
+		a.sname = ""
+		a.opts.Required = true
+		a.opts.Default = nil
+		a.size = 1 // We could allow other sizes in the future
```

Comment:
Positional is always `size == 1`, not sure in what case it wouldn't be?

Reply:
I was imagining that in the future we might use size as repetition counter for repeated positionals.